app/gui/src/model/execution_context/synchronized.rs

```diff
@@ -300,7 +300,10 @@ impl model::execution_context::API for ExecutionContext {
 
     fn restart(&self) -> BoxFuture<FallibleResult> {
         async move {
-            self.language_server.client.recompute(&self.id).await?;
+            self.language_server
+                .client
+                .recompute(&self.id, &language_server::InvalidatedExpressions::All)
```

Comment:
To keep this one line, you may import `InvalidateExpressions` or even - in scope of the function - `All`.